charts/flyte-core/values.yaml

```diff
@@ -638,15 +638,17 @@ external_events:
 cluster_resource_manager:
   # -- Enables the Cluster resource manager component
   enabled: true
-  # -- Starts the cluster resource manager in standalone mode with requisite auth credentials to call flyteadmin service endpoints
-  standalone_deploy: false
```

Comment:
Wouldn't moving this break people's existing values files, if any? is there a reason to move it?

Reply:
it's already part of the config, see: https://github.com/flyteorg/flyteadmin/blob/30862d414125eda4ef74455e4614100b522d2af8/pkg/runtime/interfaces/cluster_resource_configuration.go#L46

Reply:
and this has only existed in a beta release so far